search-commons/src/test/resources/sample_invalid_gateway_opensearch_response.json

```diff
@@ -0,0 +1,148 @@
+{
+  "statusCode": 400,
```

Comment:
is the body reflecting a 400 status code or is it copy paste?

Reply:
I was playing around with different kinds of errors, including error codes.